source/documentation/modules/meta.md

```diff
@@ -282,6 +311,31 @@ title: sass:meta
   [user-defined]: /documentation/at-rules/function
   [plain CSS function]: /documentation/at-rules/function/#plain-css-functions
 
+  The returned mixin can be included using [`meta.apply()`](#apply).
+
+  {% render 'code_snippets/example-first-class-mixin' %}
+{% endfunction %}
+
+{% function 'meta.get-mixin($name, $module: null)', 'returns:function' %}
+  {% compatibility 'dart: "1.69.0"', 'libsass: false', 'ruby: false' %}{% endcompatibility %}
+
+  Returns the [mixin value] named `$name`.
+
+  [mixin value]: /documentation/values/mixins
+
+  If `$module` is `null`, this returns the mixin named `$name` defined in the
```

Comment:
I take it that `meta.get-mixin(my-module.my-mixin)` won't return anything because it should have been written as `meta.get-mixin(my-module, my-mixin)`, is that right?

I think it might be helpful to call that out, but I realize `get-function` doesn't, so maybe that's OK.

Reply:
Let's wait and see if users actually make that mistake before we add extra text for it. I don't think I've seen anyone do that with `meta.get-function()`.